src/App.js

```diff
@@ -1,13 +1,61 @@
+import {
+  BrowserRouter as Router, Route, Routes, useLocation,
+} from 'react-router-dom';
+
+import Delete from './components/Delete/Delete';
+import Splash from './components/splash/Splash';
+import Login from './components/login/Login';
+import SignUp from './components/signup/SignUp';
+import Main from './components/main/Main';
+import Room from './components/Rooms/Room';
+import AddHotel from './components/addHotel/AddHotel';
+import Reserve from './components/Reseve/Reserve';
+import Navigation from './components/navigation/Navigation';
+import Services from './components/ourServices/Services';
+import ReservedHotel from './components/Reservation/ReservedHotel';
+import NotFound from './components/Not Found/NotFound';
 import './App.css';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this main `App` function to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the other 21 functions in your project to ES6.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://github.com/felixDev22/Final-Capstone-Frontend/assets/57421823/eab0bbfd-e59c-4b37-8dbd-219213b5359b)

That way, your code will meet the latest standards. 💪